docs/tutorials/01_introduction_cloud_runtime.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<p>It would be nice to show users how to convert this to an input they can use to other Qiskit routines.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-ibm-runtime/pull/97/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>Same as above, I don't know what the final version of <code>sampler</code>  will be and how to use it yet...</p>